assembly/src/library/error.rs

```diff
@@ -1,83 +1,27 @@
-use alloc::{string::String, vec::Vec};
-
 use vm_core::errors::KernelError;
 
-use crate::{
-    ast::QualifiedProcedureName,
-    diagnostics::Diagnostic,
-    library::{LibraryNamespaceError, VersionError},
-    prettier::pretty_print_csv,
-    DeserializationError, LibraryNamespace, LibraryPath, PathError,
-};
+use crate::{ast::QualifiedProcedureName, diagnostics::Diagnostic, LibraryNamespace, LibraryPath};
 
 #[derive(Debug, thiserror::Error, Diagnostic)]
 pub enum LibraryError {
-    #[error("library '{0}' does not contain any modules")]
+    #[error("library must contain at least one exported procedure")]
     #[diagnostic()]
-    Empty(LibraryNamespace),
-    #[error("module '{0}' not found")]
+    EmptyExports,
+    #[error("library '{0}' does not contain any modules")]
     #[diagnostic()]
-    ModuleNotFound(String),
+    EmptyModules(LibraryNamespace),
```

Comment:
Per my other related comments, I think we can probably also get rid of these two variants as well

Reply:
Removed `EmptyModules` and renamed `EmptyExports` into `EmptyKernel` as for now we require `KernelLibrary` to have at least one exported procedure.